app/presenters/individual_statistics_presenter.rb

```diff
@@ -49,51 +49,46 @@ def individual_upload_usage_count
   end
 
   def individual_article_count
-    @articles_edited.count
+    @articles_created.count
   end
 
+  # going
   def individual_article_views
-    @articles_edited.values.sum { |article| article[:views] }
+    @articles_created.values.sum { |article| article[:views] }
   end
 
   def individual_articles_created
-    @articles_edited.values.count { |article_edits| article_edits[:new_article] }
+    @articles_created.values.count { |articles| articles[:new_article] }
   end
 
   private
 
   # rubocop:disable Metrics/AbcSize
-  def set_articles_edited
-    @articles_edited = {}
+  def set_articles_created
+    @articles_created = {}
     individual_courses.each do |course|
-      individual_mainspace_edits(course).each do |edit|
-        article_edits = @articles_edited[edit.article_id] || { new_article: false,
-                                                               views: 0, characters: {},
-                                                               references: {} }
-        article_edits[:characters][edit.mw_rev_id] = edit.characters
-        article_edits[:references][edit.mw_rev_id] = edit.references_added
-        article_edits[:new_article] = true if edit.new_article
-        article_edits[:earliest_revision] = edit.date if earliest_rev_yet?(edit, article_edits)
-        article_edits[:average_views] ||= edit.article.average_views
-        @articles_edited[edit.article_id] = article_edits
+      course.articles_courses.where(new_article: true).each do |edit|
```

Comment:
I think this might not be a viable strategy. An ArticlesCourses record represents changes made to an article by (potentially) all participants in a course, not just this user.

Reply:
Thanks for the clarifications. I'd remove the condition that restricted the Article Courses to all participants in the course

Reply:
@ragesoss I've update the code implementation

Reply:
This still will count contributions from other students in the same course, not just the contributions from the student. I don't think there's a good way to use the ArticlesCourses data to get individual statistics.

Reply:
Okay, I understand now. I'd updated the function to account for only the user contribution

Reply:
@ragesoss, update! now it accounts for article contributions by the user.